drupal/apps/jobs/src/App.vue

```diff
@@ -97,16 +98,39 @@ export default {
       }
       // Sort the found jobs
       this.sortJobs();
+
+      this.detectDeeplink();
+    },
+    detectDeeplink() {
+      // Check if the url fragment (the part after #) contains a number
+      if (window.location.hash && window.location.hash.match(/^\#\d+$/)) {
```

Comment:
Why not use URL parameters?

Reply:
This feels much more like routing, so I choose to go with hashes that's normally used for routing.